src/scss/_grid-areas.scss

```diff
@@ -85,11 +85,15 @@
 @mixin _oLayoutAreaOverview() {
 	.o-layout__overview {
 		display: grid;
-		grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+		grid-template-columns: repeat(var(--_o-layout-overview-columns, auto-fit), minmax(250px, 1fr));
 		grid-gap: $_o-layout-gutter * 3;
 		grid-row-gap: $_o-layout-gutter;
 	}
 
+	.o-layout__overview--max-columns {
+		--_o-layout-overview-columns: auto-fill;
```

Comment:
Can we use CSS Variables?!

Reply:
See https://github.com/Financial-Times/o-layout/pull/95#issuecomment-508723491 :)